share/python/driver.py

```diff
@@ -463,17 +463,16 @@ def energy(name, **kwargs):
     >>> molecule H2 {\n0 1\nH\nH 1 0.74\n}
     >>> set global basis cc-pVDZ
     >>> set global reference rohf
-    >>> energy('scf')
+    >>> scf_e, scf_wfn = energy('scf', return_wfn = True)
     >>> H2.set_multiplicity(3)
-    >>> psi4.MintsHelper().integrals()
-    >>> energy('detci', bypass_scf=True)
+    >>> psi4.MintsHelper(scf_wfn.basisset()).integrals()
```

Comment:
Nevermind, apparently someone found the MintsHelper line helpful enough to deliberately add it.